app/main/views/users.py

```diff
@@ -490,3 +501,53 @@ def check_applicant_role(role, application_id):
     elif role != 'applicant' and role != 'supplier' and application_id is not None:
         abort(400, "'application_id' is only valid for users with 'applicant' or 'supplier' role, not '{}'"
               .format(role))
+
+
+@main.route('/signup/email', methods=['POST'])
+def send_signup_email():
+    try:
+        json_payload = get_json_from_request()
+        json_has_required_keys(json_payload, ['name', 'email_address', ])
+
+        name = json_payload.get('name', None)
+        email_address = json_payload.get('email_address', None)
+        employee_type = json_payload.get('employee_type', None)
+        if employee_type is None:
+            user_type = "seller"
+        else:
+            user_type = "buyer"
+
+    except ValueError as e:
+        return Response(
+            status=400,
+            headers=None
+        )
+
+    duplicate = json.loads(get_duplicate_users(email_address=email_address).data)
```

Comment:
do we need to deal with json?  you could take the core of the duplicate_users() out to a function that the route + this could call